__docs__/wonder-blocks-modal/drawer-dialog.stories.tsx

```diff
@@ -33,6 +43,55 @@ export default {
             // Drawer stories are behavior-based, disable visual regression testing
             disableSnapshot: true,
         },
+        docs: {
+            description: {
+                component: `\`DrawerDialog\` is the modal content component designed specifically for use with \`DrawerLauncher\`.
+It provides a consistent drawer interface with proper animations, positioning, and accessibility features.
+
+**IMPORTANT**: This component should only be used with \`DrawerLauncher\`. Using it with other
+modal launchers may result in incorrect animations, positioning, and styling.
```

Comment:
Good call adding this to the docs! 

Should the DrawerLauncher also only be used with DrawerDialog? Would that be helpful to document for the DrawerLauncher component too?